schema/vr.yaml

```diff
@@ -146,9 +223,70 @@ definitions:
       - $ref: "#/definitions/Allele"
       - $ref: "#/definitions/Text"
       - $ref: "#/definitions/VariationSet"
+      - $ref: "#/definitions/LocationJunction"
+    discriminator:
+      propertyName: type
+
+  MolecularVariation:
+    additionalProperties: false
+    description: >-
+      A subset of Variation describing a single resultant molecule.
+    oneOf:
+      - $ref: "#/definitions/Allele"
+      - $ref: "#/definitions/LocationJunction"
     discriminator:
       propertyName: type
 
+  Abundance:
+    additionalProperties: false
+    description: >-
+      Defines the abundance of a MolecularVariation in a systemic context. Some common
+      uses of abundance include transcript / gene expression and CNVs.
+    oneOf:
+      - $ref: "#/definitions/DiscreteCopyNumber"
+      - $ref: "#/definitions/RelativeAbundance"
+    discriminator:
+      propertyName: type
+
+  DiscreteCopyNumber:
+    additionalProperties: false
+    type: object
+    properties:
+      _id:
+        $ref: "#/definitions/CURIE"
+      type:
+        type: string
+        enum: [DiscreteCopyNumber]
+        default: DiscreteCopyNumber
+      molecularVariation:
+        $ref: "#/definitions/MolecularVariation"
+      minCopies:
+        type: integer
+      maxCopies:
+        type: integer
+    required:
+      - type
+      - molecularVariation
+
+  RelativeAbundance:
+    additionalProperties: false
+    type: object
+    properties:
+      _id:
+        $ref: "#/definitions/CURIE"
+      type:
+        type: string
+        enum: [RelativeAbundance]
+        default: RelativeAbundance
+      molecularVariation:
+        $ref: "#/definitions/MolecularVariation"
+      relativeQuantity:
+        type: string
+        enum: [greater_than, less_than, equal_to]
```

Comment:
@larrybabb suggests: We should consider abstracting this to some set of States attached to Abundance directly